app.js

```diff
@@ -1,16 +1,35 @@
 import React, {Component} from 'react'
-import Form from './Form/Form.js'
+import Nav from './Nav/Nav'
+import View from './View/View'
 
-export default class FlashCards extends Component {
+export default class App extends Component {
   constructor(props) {
     super(props)
     this.state = ({
       data: [],
       question: '',
-      answer: ''
+      answer: '',
+      showEmpty: true,
```

Comment:
You shouldn't need a `Boolean` flag for this. Either the flash cards array is empty (length of 0) or not.